src/GovUk.Education.ExploreEducationStatistics.Data.Services/AbstractTableBuilderSubjectMetaService.cs

```diff
@@ -39,6 +39,27 @@ protected AbstractTableBuilderSubjectMetaService(IFilterItemService filterItemSe
                         TotalValue = GetTotalValue(itemsGroupedByFilter)
                     });
         }
+        
+        protected Dictionary<string, FilterMetaViewModel> GetFilters2(IQueryable<Observation> observations)
```

Comment:
Is this used because the code seems to be identical to the method GetFilters

Reply:
Great spot - removed!